api/app/views/mno_enterprise/jpi/v1/current_users/show.json.jbuilder

```diff
@@ -31,14 +31,17 @@ json.cache! ['v2', @user.cache_key] do
     # Embed association if user is persisted
     if @user.id
       json.organizations do
-        json.array! (@user.organizations.select(&:active?) || []) do |o|
-          json.id o.id
-          json.uid o.uid
-          json.name o.name
-          json.currency o.billing_currency
-          json.current_user_role @user.role(o)
-          json.has_myob_essentials_only o.has_myob_essentials_only
-          json.financial_year_end_month o.financial_year_end_month
+        json.array! @user.organizations do |o|
```

Comment:
I think it would be worth doing the filtering on the controller.
I would suggest to stop using the organization include in current_user (which is bringing issue as it's not paginated anyway) and introduce `@organizations = MnoEnterprise::Organization.where('users.id': current_user.id) and then filter the array on the .visible? method 
Or better, find a way to introduce 'OR' operator in MnoHub Api v2.